unit_tests/node_kernels/UnitTestMomentumGclSrcNodeKernel.C

```diff
@@ -0,0 +1,50 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "kernels/UnitTestKernelUtils.h"
+#include "UnitTestUtils.h"
+#include "UnitTestHelperObjects.h"
+
+#include "node_kernels/MomentumGclSrcNodeKernel.h"
+#include "node_kernels/UnitTestNodeUtils.h"
+
+#include <vector>
+
+TEST_F(MomentumNodeHex8Mesh, NGP_momentum_gcl_node)
+{
+  // Only execute for 1 processor runs
+  if (bulk_.parallel_size() > 1) return;
+
+  fill_mesh_and_init_fields();
+
+  sierra::nalu::TimeIntegrator timeIntegrator;
+  timeIntegrator.timeStepN_ = 0.1;
+  timeIntegrator.timeStepNm1_ = 0.1;
+  timeIntegrator.gamma1_ = 1.0;
+  timeIntegrator.gamma2_ = 0.0;
+  timeIntegrator.gamma3_ = 0.0;
+
+  unit_test_utils::NodeHelperObjects helperObjs(
+    bulk_, stk::topology::HEX_8, 3, partVec_[0]);
+
+  helperObjs.realm.timeIntegrator_ = &timeIntegrator;
+
+  helperObjs.nodeAlg->add_kernel<sierra::nalu::MomentumGclSrcNodeKernel>(bulk_);
+
+  helperObjs.print_lhs_and_rhs();
```

Comment:
Swap this line with next to get the proper `rhs` values. They should not be zero.